rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -2229,7 +2229,7 @@ private String name() {
     }
 
     /*
-       Visit the value filled in to a parameterized type, such as in a variable declaration.
+       Visit the value filled into a parameterized type, such as in a variable declaration.
```

Comment:
`in to` can sometimes be correct, e.g. `filled in to make something happen`, but, generally if the word after `in?to` isn't a verb, it probably shouldn't be `in to`.